delta/middleman.py

```diff
@@ -22,26 +20,40 @@
 import timeit
 import time
 
-from streaming.reader_nompi import reader_gen
-from streaming.writer_nompi import writer_gen
-from analysis.channels import channel_range
-
+from sources.dataloader import get_loader
+from streaming.writers import writer_gen
+from streaming.reader_mpi import reader_gen
+from data_models.helpers import gen_channel_name, gen_var_name
 
 @attr.s
 class AdiosMessage:
```

Comment:
Let's get rid of AdiosMessage and use ecei_chunk as a wrapper everywhere.

Reply:
I removed get_loader. Others are needed. I pushed a new commit.